x-pack/platform/plugins/shared/streams/server/lib/streams/state_management/streams/group_stream.ts

```diff
@@ -77,10 +77,11 @@ export class GroupStream extends StreamActiveRecord<Streams.GroupStream.Definiti
         definition: {
           name: member,
           description: '',
+          updated_at: new Date().toISOString(),
           ingest: {
             classic: {},
             lifecycle: { inherit: {} },
-            processing: { steps: [] },
+            processing: { steps: [], updated_at: new Date().toISOString() },
```

Comment:
Same here as in https://github.com/elastic/kibana/pull/242904/files#r2585201629, please also check it anywhere else in the PR